docs/en/reference/working-with-associations.rst

```diff
@@ -616,11 +616,11 @@ address reference. When flush is called not only are the references removed
 but both the old standing data and the one address entity are also deleted 
 from the database.
 
+.. _filtering-collections:
+
 Filtering Collections
 ---------------------
 
-.. filtering-collections:
```

Comment:
I believe the `_` was left off accidentally before - I'm not familiar with this syntax without an underscore, and I searched the codebase and found NO other references like this (`git grep '\.\. ' | grep -v 'code-block' | grep -v 'note' | grep -v 'section' | grep -v '\.\. _' | grep -v 'configuration-block' | grep -v versionadded | grep -v 'warning' | grep -v '//' | grep -v '\-\->' | grep -v 'include' | grep -v 'caution' | less` gets you a short list, where you can see this). Also, this section header already gave us a `#filtering-collections` anchor, even without the added reference.